templates/resources/index.html

```diff
@@ -0,0 +1,405 @@
+        <p>Watch it <a href="https://ubuntu.com/engage/self-healing-edge-kubernetes-webinar">here</a>.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/engage/container-orchestration-edge">
+            <img src="https://www.brighttalk.com/communication/508196/preview_1631023307.png" alt="Self-healing Kubernetes at the edge: MicroK8s, Raspberry Pis and Portainer" width="560" height="302">
+           </a>
+        </div>
+      </div>
+    </div>
+  </article>
+  <hr class="is-muted is-fixed-width" />
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4">MicroK8s on IBM Z &mdash; minimal footprint meets zero downtime</h3>
+      </div>
+      <div class="col-4">
+        <p>Mission-critical applications require resilient Kubernetes that can lose any node and still provide reliable services.</p>
+        <p>This webinar presents the case for running MicroK8s on IBM Z/Linux ONE/s390x systems: combining a robust, low-touch Kubernetes with the most reliable hardware platform available.</p>
+        <p>Watch it <a href="https://ubuntu.com/engage/webinar-microk8s-on-z">here</a>.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/engage/container-orchestration-edge">
+            <img src="https://assets.ubuntu.com/v1/3a72e1ce-k8s+Microk8s+ibmz.png" alt="MicroK8s on IBM Z - minimal footprint meets zero downtime" width="560" height="302">
+           </a>
+        </div>
+      </div>
+    </div>
+  </article>
+</section>
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Case studies/Whitepapers/Industry reports</h2>
+  </div>
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4">Kubernetes: a secure, flexible and automated edge for IoT developers</h3>
+      </div>
+      <div class="col-4">
+        <p>This 451 Research whitepaper focuses on the key factors that drive the evolution of IoT edge and the requirements around which Kubernetes need to optimise to become the standard edge platform.</p>
+        <p>MicroK8s, being a low-touch, low-footprint Kubernetes distribution, already addresses many of the key essentials!</p>
+        <p>Learn more <a href="https://ubuntu.com/engage/microk8s-451research">here</a>.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/engage/container-orchestration-edge">
+            <img src="https://assets.ubuntu.com/v1/3e64aa42-Canonical_451Research_MicroK8s_Final-1.png" alt="MicroK8s on IBM Z" width="560" height="302" style="min-height: 183px">
```

Comment:
This image is rendering a bit weird (looks a bit squished and kind of blurry), but I'm not sure if its just me? Feel free to disregard if it looks ok to you!